### PR TITLE
Fixed a broken link

### PR DESCRIPTION
**Pull request by waseem18**

### CODE REVIEW

**Comment by bryevdv** [comment]
Hi @waseem18 thanks for the good catch. It looks like the gallery link needs to be updated in the same way, also I think it explicitly needs to be `http://` can you make those changes as well, as long as you are working on this file?

**Comment by waseem18** [comment]
sure @bryevdv I'll make the changes and report it soon.

**Comment by waseem18** [comment]
Made required changes @bryevdv

File: content/pages/about.md [hunk 1]
```diff
@@ -12,10 +12,10 @@ the Technical Vision.
 
 
 To get started quickly, follow the
-[Quickstart](//bokeh.pydata.org/en/latest/user_guide/quickstart.html).
+[Quickstart](http://bokeh.pydata.org/en/latest/docs/user_guide/quickstart.html#userguide-quickstart).
```
Comment (bryevdv) [inline content/pages/about.md]: I don't think the anchor is necessary, just a correct link to `quickstart.html`

**Comment by bryevdv** [comment]
One minor comment, otherwise LGTM

**Comment by waseem18** [comment]
Updated @bryevdv

**Comment by bryevdv** [comment]
thanks!

**Pull request merged**